netbox/extras/tests/test_views.py

```diff
@@ -119,6 +120,31 @@ def setUpTestData(cls):
             'description': 'New description',
         }
 
+    # These are here as extra_choices field splits on colon, but is returned
+    # from DB as comma separated.  So we exclude them in validation_excluded_fields
+    # but test they are actually set correctly here.
```

Comment:
It might be cleaner to override `assertInstanceEqual()` on the test class, and overwrite `data['extra_choices`] there (before calling `super()`).